content/en/getting_started/test_impact_analysis/_index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Getting Started with Test Impact Analysis
+aliases:
+- /getting_started/intelligent_test_runner
+further_reading:
+- link: 'https://www.datadoghq.com/blog/streamline-ci-testing-with-datadog-intelligent-test-runner/'
+  tag: 'Blog'
+  text: 'Streamline your CI testing with Datadog Intelligent Test Runner'
```

Comment:
Likewise, here we refer to a document with the old name, and it might not be clear to a customer that this is the same feature.

Reply:
Thanks @vitor-de-araujo for the callout, I'm going to contact the Writing team to see if we can update the title of these blogs, and then we can update them on the docs accordingly.

Reply:
Created [WRIT-1109](https://datadoghq.atlassian.net/browse/WRIT-1109) and will update docs when the blog is updated. Thanks!